py-polars/tests/unit/interop/numpy/test_ufunc_series.py

```diff
@@ -1,8 +1,26 @@
 from typing import cast
 
 import numpy as np
+import pytest
 from numpy.testing import assert_array_equal
 
+try:
+    from numba import float64, guvectorize  # type: ignore[import-untyped]
```

Comment:
This is more of a note to self but if we're putting numba in the tests then there should be some tests for the pl.Expr version of `__array_ufunc__` too.

Reply:
I can add a couple in this PR.